Classes/Private/Manager/OCTManager.m

```diff
@@ -27,6 +28,7 @@ @interface OCTManager () <OCTToxDelegate, OCTSubmanagerDataSource>
 @property (strong, nonatomic, readwrite) OCTSubmanagerChats *chats;
 @property (strong, nonatomic, readwrite) OCTSubmanagerAvatars *avatars;
 @property (strong, nonatomic, readwrite) OCTSubmanagerFiles *files;
+@property (strong, nonatomic, readwrite) OCTSubmanagerCalls *managerCalls;
```

Comment:
Well, we have OCTFriend and `friend` property. :-)
From user prospective it would look like `manager.friends` or `manager.calls`, which is quite okay.

So yeah, I think you can just change it to `calls`.